src/js/main-window.js

```diff
@@ -6,9 +6,13 @@ var Accel = require('ui/accel');
 var Utils = require('utils');
 var Menu = require('menu');
 var Elements = require('elements');
+var Feature = require('platform/feature');
+var Settings = require('settings');
+
+Accel.init();
 
 var mainWindow = new UI.Window({
-  fullscreen: true, 
```

Comment:
It's giving a warning about deprecation, but they don't say that in documentation: https://pebble.github.io/pebblejs/#window
Also, there is no other way of taking fullscreen, and because our main window mimics the thermostat screen, bar on the top would simply look weird.
I'd leave this as is for now until we know how to take fulls creen without using deprecated code.